_posts/2019-11-01-deploying-debezium-with-kafkaconnector-resource.md

```diff
@@ -0,0 +1,697 @@
+---
+layout: post
+title:  "Deploying Debezium using the new KafkaConnector resource"
+date: 2019-11-01
+author: tom_bentley
+---
+
+For too long our Kafka Connect story hasn't quite as "Kubernetes-native" as it could have been.
+We had a `KafkaConnect` resource to configure a Kafka Connect _cluster_ but you still had to use the Kafka Connect REST API to actually create a _connector_ within it.
+While this wasn't especially difficult using something like `curl`, it stood out because everything else could be done using `kubectl` and it meant that connectors didn't fit into our Kubernetes-native vision.
+With the help of a contribution from the community, Strimzi now supports a `KafkaConnector` custom resource and the rest of this blog post is going to explain how to use it using [Debezium](https://debezium.io) as an example. 
+
+And as if that wasn't enough, there's some awesome ASCII art.
```

Comment:
I don't quite get this one.

Reply:
Yeah, it's a weak attempt to add a little humour. Did you appreciate the joke in the end (maybe I need to tweak this first sentence to make it work better)? Or did you think it was too lame to be worth it?

Reply:
Yep, got it in the end and liked it. It won't nominate you for the comedy Oscars, but it's not lame either. It was just a bit contextless here, perhaps make some forward-looking link to qualify it a bit. Perhaps "How so? Read on to find out" or similar would do the trick.